src/main/java/ay2122s1_cs2103t_w16_2/btbb/model/order/Order.java

```diff
@@ -87,10 +88,10 @@ public int hashCode() {
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
-        builder.append("Phone: ")
-                .append(getClientPhone())
-                .append(" Name: ")
+        builder.append(" Name: ")
```

Comment:
```suggestion
        builder.append(" Client Name: ")
```

Similarly for `Client Phone` and `Client Address`